src/linker/Linker.Dataflow/FlowInsensitiveReflectionScanner.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using ILLink.Shared.DataFlow;
+using ILLink.Shared.TrimAnalysis;
+using Mono.Cecil;
+using Mono.Linker.Steps;
+using MultiValue = ILLink.Shared.DataFlow.ValueSet<ILLink.Shared.DataFlow.SingleValue>;
+
+namespace Mono.Linker.Dataflow
+{
+	public class FlowInsensitiveReflectionScanner
```

Comment:
Very odd name. If the type does not need DataFlow why is it in the namespace?

Reply:
I actually don't mind this being in the DataFlow - it is data flow, just not inside a method body.
Maybe we should split this further:
* `AttributeDataFlow` - handle all of the attribute stuff - we could even move some of the code from MarkStep to here possibly (NativeAOT has lot more of the code in these methods as well).
* `GenericArgumentDataFlow` - just the generic stuff - it's going to be small, but so what.
* And maybe we should then rename the `ReflectionMethodBodyScanner` to `MethodBodyDataFlow` - or something like that. But I'm not sure about this - both names have downsides as well.